### ISSUE TITLE
Looking for global config when local is present

### ISSUE THREAD

**Issue by chadfawcett** [eslint/eslint#10287]
**Tell us about your environment**

* **ESLint Version:** 4.19.1
* **Node Version:** 8.11.1
* **npm Version:** 6.0.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
    "extends": "standard"
};
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
// index.js
console.log('testing');
```

```bash
npm install --save eslint
npx eslint --init
# Use a popular style guide
# Standard
# Javascript
npx eslint index.js
```

**What did you expect to happen?**
I expect eslint to use local configuration file and output any errors/warnings.

**What actually happened? Please include the actual, raw output from ESLint.**
eslint is looking at the root of my user directory. If there is an error in the configuration file, I will get an error message telling me so. For that reason, I can tell eslint **does** see my local config file. I'm not sure why it then looks for one in my user directory.
```
ENOENT: no such file or directory, stat '/Users/chadfawcett/.eslintrc.js'
```

**Comment by chadfawcett** [comment]
I accidentally had a broken symlink in my root directory names `~/.eslintrc.js`. I removed the broken symlink and everything works as expected.
